Venmo/src/main/java/com/braintreepayments/api/VenmoInternalCallback.java

```diff
@@ -0,0 +1,16 @@
+package com.braintreepayments.api;
+
+import androidx.annotation.Nullable;
+
+/**
+ * Callback for receiving result of
+ * {@link VenmoClient#tokenize(VenmoPaymentAuthResult, VenmoTokenizeCallback)}.
+ */
+interface VenmoInternalCallback {
```

Comment:
Internal and external methods shared the same callback for delivering (nonce, exception) results - I opted to only refactor the public facing callbacks for now, so kept this one and renamed it to internal. Future code cleanup will be to migrate all internal code to use the single result object pattern too.